internal/handlers/lokistack_create_or_update.go

```diff
@@ -63,13 +63,13 @@ func CreateOrUpdateLokiStack(ctx context.Context, req ctrl.Request, k k8s.Client
 
 	// Here we will translate the lokiv1beta1.LokiStack options into manifest options
 	opts := manifests.Options{
-		Name:                        req.Name,
-		Namespace:                   req.Namespace,
-		Image:                       img,
-		Stack:                       stack.Spec,
-		ObjectStorage:               *storage,
-		UseServiceMonitors:          manifests.UseServiceMonitors,
-		UseTLSEnabledServiceMonitor: manifests.UseTLSEnabledServiceMonitors,
+		Name:                          req.Name,
+		Namespace:                     req.Namespace,
+		Image:                         img,
+		Stack:                         stack.Spec,
+		ObjectStorage:                 *storage,
+		EnableServiceMonitors:         manifests.UseServiceMonitors,
+		EnableTLSServiceMonitorConfig: manifests.UseTLSEnabledServiceMonitors,
```

Comment:
Would you mind to also rename the right hand side to `Enable..` instead of `Use...`

Reply:
Yeah, I haven't touched these ones yet as I'm still looking at a way of nicely accomplishing this: https://github.com/ViaQ/loki-operator/pull/59#discussion_r649726800